listenbrainz/webserver/static/css/follow.less

```diff
@@ -0,0 +1,73 @@
+@purple: #353070;
+@white: #ffffff;
+
+#follow-button {
+  margin-left: 10px;
+  font-family: "Sintony";
+
+  &.icon-only {
+    > .text {
+      margin-left: 4px;
+      width: 0px;
+      display: inherit;
+      vertical-align: inherit;
+      transition: width 0.42s;
+      overflow-x: hidden;
+    }
+    &:hover > .text,
+    &:focus > .text {
+      width: 4.5em;
+    }
+  }
+
+  &.block {
+    color: @purple;
+    background: @white;
+    border: 2px solid @purple;
+    font-weight: bold;
+    border-radius: 6px;
+    padding: 5px;
+    text-align: center;
+    width: 85px;
+
+    &.btn-info {
+      color: @white;
+      background: @purple;
+    }
+
+    svg {
+      display: none;
+    }
+  }
+}
+
+.follower-following-pills {
+  padding-bottom: 10px;
+}
+
+.follower-following-list {
+  height: 250px;
+  padding: 10px;
+  overflow-y: scroll;
+  border-radius: 2px;
+  box-shadow: inset 0px 11px 8px -10px #ccc;
+
+  > div {
+    display: flex;
+    align-items: center;
+    padding: 5px;
+    margin: 5px 10px;
+
+    > #follow-button {
+      margin-left: auto;
+
+    }
+  }
+  hr {
```

Comment:
To me it's a bit weird to style a horizontal rule like that.
I can propose an alternative with pseudo elements:
```css
.follower-following-list {
  [...]
  > div {
  [...]
    &:not(:first-child):before {
      content : "";
      position: absolute;
      margin-bottom: 50px;
      width   : 80%;
      border: 1px solid #ddd;
      border-radius: 2px;
    }
  }
}
```

Reply:
The pseudo element remains fixed and doesn't scroll. Any idea how to fix that?

Reply:
Hmm, that rings a bell, but I don't remember exactly how I solved it.
I must have forgotten a css property on the pseudo-element or its parent, maybe something to do with `position`.

I'll try to run it again.

Reply:
It's working fine for me, and scrolling along with the rest of the page. 

![following](https://user-images.githubusercontent.com/6179856/99649427-7dbba100-2a54-11eb-8c57-4b059e587f57.gif)

Here's that full .less segment, in case that's the issue:

```less
.follower-following-list {
  height: 250px;
  padding: 10px;
  overflow-y: scroll;
  border-radius: 2px;
  box-shadow: inset 0px 11px 8px -10px #ccc;

  > div {
    display: flex;
    align-items: center;
    padding: 5px;
    margin: 5px 10px;

    > #follow-button {
      margin-left: auto;

    }
    &:not(:first-child):before {
      content : "";
      position: absolute;
      margin-bottom: 50px;
      width   : 80%;
      left: 10%; // to center, should be half of what is removed from full width
      border: 1px solid #ddd;
      border-radius: 2px;
    }
  }
}
```